sse2neon.h

```diff
@@ -1000,33 +1000,13 @@ FORCE_INLINE __m128i _mm_shuffle_epi_3332(__m128i a)
 
 // Shuffle packed 8-bit integers in a according to shuffle control mask in the
 // corresponding 8-bit element of b, and store the results in dst.
-FORCE_INLINE __m128i _mm_shuffle_epi8(__m128i ia, __m128i ib)
+// https://software.intel.com/sites/landingpage/IntrinsicsGuide/#text=_mm_shuffle_epi8&expand=5146
+FORCE_INLINE __m128i _mm_shuffle_epi8(__m128i a, __m128i b)
 {
-    uint8_t *a = (uint8_t *) &ia;  // input a
-    uint8_t *b = (uint8_t *) &ib;  // input b
-    int32_t r[4];
+    int8x16_t t = vreinterpretq_s8_m128i(a);     // input a
+    uint8x16_t idx = vreinterpretq_u8_m128i(b);  // input b
```

Comment:
Can you replace `t` and `idx` with something mnemonic?

Reply:
I just follow the input argument naming by searching the `vqtbl1q_s8` in the [Neon intrinsic page](https://developer.arm.com/architectures/instruction-sets/simd-isas/neon/intrinsics).

I think the `t` variable can be changed to `tbl`, or remains the same to follow the description in the Neon intrinsic page.

Reply:
`tbl` sounds better since it is a matter of table lookup operations.

Reply:
Done.